Common/SimConfig/src/SimConfig.cxx

```diff
@@ -48,6 +48,7 @@ void SimConfig::initOptions(boost::program_options::options_description& options
     "chunkSize", bpo::value<unsigned int>()->default_value(5000), "max size of primary chunk (subevent) distributed by server")(
     "chunkSizeI", bpo::value<int>()->default_value(-1), "internalChunkSize")(
     "seed", bpo::value<int>()->default_value(-1), "initial seed (default: -1 random)")(
+    "field", bpo::value<int>()->default_value(-5), "L3 field rounded to kGauss, allowed values +-2,+-5 and 0")(
```

Comment:
I have a slight preference in using the ConfigurableParams for this. In particular since you have well defined allowed values which could be described by enums and written out as text in the configuration.

Reply:
@sawenzel Where would you put it? The field is one of the parameters which usually should not be set by hand but taken from the anchored run GRP (eventually from the CCDB). I think it is better to keep it as a separate option to not mix if with parameters which should be always set.

Reply:
The ConfigurableParams are actually thought to be coming from CCDB/GRP finally (which speaks even more in their favour). We could do a GRPParam or a FieldParam.

However, I have nothing against merging this now if it unblocks some important tests. (We can continue discussing offline.)

Reply:
@sawenzel to do this cleanly, we have to approach from the opposite side: always start the simulation from some GRP (rather than generate it from input parameters). But this would require quite some changes, let's add this option at the moment as is (the o2 test failure is unrelated).